### ISSUE TITLE
Arrow isn't clickable

### ISSUE THREAD

**Issue by mgibbs189** [mgibbs189/fSelect#10]
When you click in the box the dropdown appears. But if you click on the span with the class "fs-arrow" (the down triangle that appears in the control) the dropdown does not appear/disappear. (A more precise click location is at the top middle of the black down triangle if you move your mouse pointer down one or two pixels, the dropdown doesn't toggle.)

**Comment by daminloi** [comment]
I think I have a fix using css and an image for the down arrow, at least it works for me.

Hide the fs-arrow class span.

```
span.fs-arrow {
display:none;
}

```
Then apply a background image of the down arrow to the .fs-label-wrap class.

```
.fs-label-wrap {
background: url(http://i.imgur.com/9Rgx7at.jpg) 96% / 5% no-repeat;
}
```

This got around that odd "dead" spot that clicking on doesn't toggle the dropdown list. (Recommended you host the down arrow image from imgur on your site, if you want to use this temporary fix.)

**Comment by mgibbs189** [comment]
Thanks @daminloi. So you haven't had any luck with `.fs-arrow` itself?

**Comment by daminloi** [comment]
No. I tried to add an on click event for elements with the `.fs-arrow` class itself (basically a copy paste of the function starting at line 196 of `fselect.js`, except using `.fs-arrow` instead of `document`) but it didn't seem to do anything. I could trigger a test alert popup I inserted, but it wouldn't trigger the dropdown. Maybe it was triggering something, but it created a conflict with the part of your code that deals with the on click of the document element, so one cancelled the other out? I am by no means a jQuery/javascript expert so I could very well have missed something when I tried this.
